libs/transactions/src/lib/mocks/transactions-mocks.ts

```diff
@@ -1,96 +1,71 @@
-import { Transaction } from '../model/transaction';
+import { TransactionItem } from '@backbase/data-ang/transactions';
 
-export const transactionsMock: Transaction[] = [
+export const transactionsMock = [
```

Comment:
Why do we still need these mocks?

Reply:
For unit tests, they help to avoid duplication